### ISSUE TITLE
Mouse engine settings

### ISSUE THREAD

**Issue by ryan980** [fqlx/XboxKeyboardMouse#118]
So with a controller, there is usually a deadzone put in place. Now try using a mouse that has a deadzone. When you move the mouse, nothing happens. You have to keep moving it in that direction to bypass the deadzone and be able to move. That's a pain. I tried switching it to raw input and nothing happens now. The move will move when it's set to the default mouse engine (deadzone), but when you put the mouse engine on raw input, nothing happens when you move your mouse. I haven't tried any of the other options yet, and I'm sure you are well aware of the issue, but I am just here to help.

**Comment by diablo2002** [comment]
Yeah , you are right & also the mouse isn't that bad bad but not that good either in shooter games like rainbow six siege when a pixel matter it doesn't feel like you are playing with an adapter

**Comment by DavidRieman** [comment]
"try using a mouse that has a deadzone" - do you mean using a mouse that comes with software that nullifies small mouse movements, like, _on purpose_?  Is that a thing?

After re-reading with a potential different interpretation of what you wrote:  The dead zone mouse engine is built to _compensate_ for controller dead zones by translating 1 pixel of mouse movement into just-past-the-controller-dead-zone controller input.  Once calibrated, even the smallest mouse movements are supposed to translate into small aim movements.  But you have to calibrate with the F12/F11 keys per the github readme/instructions because every game's programmed dead zone can be different.  So I interpret maybe you missed the calibration steps.
